agrirouter-sdk-java-impl/src/main/java/com/dke/data/agrirouter/impl/messaging/encoding/EncodeMessageServiceImpl.java

```diff
@@ -106,7 +106,7 @@ public String encode(
     messageHeaderParameters.validate();
     payloadParameters.validate();
 
-    if (messageHeaderParameters.getTechnicalMessageType().getNeedsChunking()) {
+    if (messageHeaderParameters.getTechnicalMessageType().needsBase64Encoding()) {
```

Comment:
I would like to add a comment here, e.g.
```
//Data that needs Base64Encoding also needs chunking. That's what we are checking here
if (messageHeaderParameters.getTechnicalMessageType().needsBase64Encoding()) {
```

Reply:
Inline comments are against clean code rules >>> http://www.laputan.org/pub/patterns/fowler/smells.pdf

I will find a better solution.